code/game/machinery/recycler.dm

```diff
@@ -111,17 +111,19 @@
 	SIGNAL_HANDLER
 	INVOKE_ASYNC(src, PROC_REF(eat), AM)
 
-/obj/machinery/recycler/proc/eat(atom/movable/AM0, sound=TRUE)
+/obj/machinery/recycler/proc/eat(atom/movable/morsel, sound=TRUE)
 	if(machine_stat & (BROKEN|NOPOWER))
 		return
 	if(safety_mode)
 		return
-	if(iseffect(AM0))
+	if(iseffect(morsel))
 		return
-	if(!isturf(AM0.loc))
+	if(!isturf(morsel.loc))
 		return //I don't know how you called Crossed() but stop it.
+	if(morsel.resistance_flags & INDESTRUCTIBLE)
```

Comment:
Should we call ``emergency_stop()`` here like we do when recycling a human?

Reply:
We certainly could for consistency's sake. The difference basically is whether or not we want it to not eat anything if an invalid object enters, or if we just want it ignored by the machine.

Reply:
Doesn't have to be.